doc/userguide/rules/datasets.rst

```diff
@@ -96,6 +96,23 @@ hashsize <size>
 
 .. note:: 'load' and 'state' or 'save' and 'state' cannot be mixed.
 
+Example rules could look like:
+
+1. Detect unique User-Agents::
+
+    alert http any any -> any any (msg:"LOCAL HTTP new UA";http.user_agent;dataset:set,http-ua-seen, type string, state http-ua-seen.csv; sid:8000001; rev:1;)
```

Comment:
Maybe we could use the rule container? (`.. container:: example-rule`)

Reply:
(very) nit: space after `msg` semi-colon

Reply:
Didn't know about the rule container. Thanks! :)

Reply:
There's always something new to learn xD yw!

Reply:
@jufajardini sorry for coming back late at this but do you want space after every semi-colon separated field or just the `msg`?

Reply:
Good point! Whenever it happens (except for the parenthesis case :P)
It increases readability for human folks, imho.

Reply:
nvm there aren't any more